lib/global-styles.php

```diff
@@ -708,7 +743,7 @@ function gutenberg_experimental_global_styles_settings( $settings ) {
 	// STEP 1: ADD FEATURES
 	// These need to be added to settings always.
 	// We also need to unset the deprecated settings defined by core.
-	$settings['__experimentalFeatures'] = gutenberg_experimental_global_styles_get_editor_features( $merged );
+	$settings['__experimentalFeatures'] = gutenberg_experimental_global_styles_get_editor_settings( $merged );
```

Comment:
I guess `__experimentalFeatures` doesn't make sense anymore? Maybe ultimately we should just merge `gutenberg_experimental_global_styles_get_editor_settings( $merged );` into `$settings` (when we make them stable)

Reply:
Yah as I said in https://github.com/WordPress/gutenberg/pull/25301#discussion_r488020736 I'm not sure about the best name. __experimentalSettings would be strange as when things become stable we would have a `{ settings: { settings: ... } }`object. Merging into $settings also does not seems like a good bet we would have `{ settings: { availableLegacyWigets: ..., currentSettingA: a, global: { color: {...} }, 'core/paragraph': { color: {...} }, 'third/block...': { color: {...} } `

Reply:
Now that presets are present in `__experimentalFeatures` I don't think they should be in `__experimentalGlobalStylesBaseStyles` as well (`settings` can be removed from the later so base styles only include the core&theme styles).

Reply:
If we rename `__experimentalFeatures` it feels like we should also rename the `useEditorFeature` hook. Throwing out some ideas: how about `__experimentalBlockSettings` and `useBlockSetting`? I'm also fine keeping things as they are until they're stabilized, though.